frontend/src/components/LandingPage/Tools/ToolCard.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+import { Card, CardBody, CardSubtitle, CardTitle, CardText } from 'reactstrap';
+
+function ToolCard(props) {
+    return (
+    <Card style={{backgroundColor:"#eee", borderColor:"#333", margin:".5%"}}>
```

Comment:
What do you think about creating a stylesheet and importing it instead of using inline styling?